fastq_filtrator.py

```diff
@@ -0,0 +1,152 @@
+
+def read_func(input_path):
+    with open(input_path, "r") as file:
+        data = file.read()
+    return data
+
+
+def get_every_read(data):
+    reads = []
+    data = data.split("\n")     # Разбиваем строку по переносу строки, получаем список.
+    start_idx = 0
+    for i in range(4, len(data), 4):    # Каждый  read чередуется с периодичностью 4.
```

Comment:
В текущем варианте пропускается последний рид
```suggestion
    for i in range(4, len(data) + 1, 4):    # Каждый  read чередуется с периодичностью 4.
```